ui/raidboss/data/triggers/o8s.js

```diff
@@ -519,8 +567,8 @@
       replaceSync: {
         'Graven Image': 'Heilige Statue',
         'Kefka': 'Kefka',
-        'Light Of Consecration': 'Licht[p] Der Weihe',
-        'The Mad Head': 'Verrückt[a] Kopf',
+        'Light Of Consecration': 'Licht Der Weihe',
+        'The Mad Head': 'Verrückter Kopf',
```

Comment:
Thanks for fixing these up.  I wasn't sure what to do with this, because this is what fflogs gave me back.

Reply:
Yup, that's a common mistake ff14angler, xivdb and fflogs do, but in the game and in the german logs it's usually the correct expression. I've checked it from my last o8s log :)